charts/tidb-cluster/templates/scripts/_initialize_tidb_users.py.tpl

```diff
@@ -1,7 +1,22 @@
-import os, MySQLdb
+import os, sys, MySQLdb
 host = '{{ template "cluster.name" . }}-tidb'
 permit_host = {{ .Values.tidb.permitHost | default "%" | quote }}
 port = 4000
+{{- if .Values.tidb.passwordSecretName }}
```

Comment:
The chart `tidb-cluster` is not used since TiDB Operator v1.1.
Please also help update the code here https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/template.go#L282.

Reply:
ok will do.